views/download.handlebars

```diff
@@ -35,5 +35,5 @@
     </div>
   </div>
 
-  <a class="send-new" data-l10n-id="sendYourFilesLink" target="_blank"></a>
+  <a class="send-new" data-l10n-id="sendYourFilesLink" href="/"></a>
```

Comment:
See https://github.com/mozilla/send/blob/master/frontend/src/links.js#L7-L19 which gets called at the start of a download.